src/js/components/modal.js

```diff
@@ -136,6 +148,18 @@ class Modal {
     this.bindTrigger()
     this.fillModal()
   }
+
+  getCloseTemplate () {
+    if (this.isStaticModal()) {
+      return $('')
```

Comment:
Or like I said on the Slack channel, you could use `if` statement on the `bindListeners ` method. If the modal is `static`, simple don't register the `click` event on the `$close` element.

What do you think about this approach?